test/util.go

```diff
@@ -15,15 +15,16 @@ func Tap(obj fyne.Tappable) {
 }
 
 // TapSecondary simulates a right mouse click on the specified object.
-func TapSecondary(obj fyne.Tappable) {
+func TapSecondary(obj fyne.Tappable) (ev *fyne.PointEvent) {
 	if focus, ok := obj.(fyne.Focusable); ok {
 		if focus != Canvas().Focused() {
 			Canvas().Focus(focus)
 		}
 	}
 
-	ev := &fyne.PointEvent{Position: fyne.NewPos(1, 1)}
+	ev = &fyne.PointEvent{Position: fyne.NewPos(1, 1)}
```

Comment:
This pattern does not seem quite correct... Could we add TapSecondaryAt(fyne.Tappable, fyne.Position) instead? that would better simulate the API usage and would put the test in control of what it was testing...

Reply:
My bad. I agree that'll be better.